streams/src/main/java/org/apache/kafka/streams/kstream/Windows.java

```diff
@@ -24,7 +24,7 @@
 import java.util.Objects;
 
 /**
- * The window specification interface for fixed size windows that is used to define window boundaries and grace period.
+ * The window specification class for fixed size windows that is used to define window boundaries and grace period.
```

Comment:
Can we agree on `The window specification for fixed...` ?

It seems unnecessary to redundantly state that it is a class (although it's surely better than saying it's an interface).